ansible-fmw/roles/dojo-wls-lifecycle/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
plz kindly update README , this content is autogenerated ;)

Reply:
We have deleted the blank role.